oak/server/rust/oak_runtime/src/node/wasm/mod.rs

```diff
@@ -537,113 +530,134 @@ impl wasmi::Externals for WasmInterface {
 }
 
 impl wasmi::ModuleImportResolver for WasmInterface {
-    /// A resolver function, mapping function names to an index and a type signature.
     fn resolve_func(
         &self,
         field_name: &str,
         signature: &wasmi::Signature,
     ) -> Result<wasmi::FuncRef, wasmi::Error> {
-        // The types in the signatures correspond to the parameters from
-        // /oak/server/rust/oak_abi/src/lib.rs
-        let (index, sig) = match field_name {
-            "node_create" => (
-                NODE_CREATE,
-                wasmi::Signature::new(
-                    &[
-                        ABI_USIZE,      // config_buf
-                        ABI_USIZE,      // config_len
-                        ABI_USIZE,      // entrypoint_buf
-                        ABI_USIZE,      // entrypoint_len
-                        ABI_USIZE,      // label_buf
-                        ABI_USIZE,      // label_len
-                        ValueType::I64, // handle
-                    ][..],
-                    Some(ValueType::I32),
-                ),
+        oak_resolve_func(field_name, signature)
+    }
+}
+
+/// A resolver function, mapping Oak host function names to an index and a type signature.
+fn oak_resolve_func( field_name: &str, signature: &wasmi::Signature, ) -> Result<wasmi::FuncRef, wasmi::Error> {
```

Comment:
Not sure what's going on with the formatting.

Reply:
Fixed (something up with my auto-format save-hook)